platform/ios/src/MGLMapView.mm

```diff
@@ -444,6 +449,7 @@ - (void)commonInit
 {
     _isTargetingInterfaceBuilder = NSProcessInfo.processInfo.mgl_isInterfaceBuilderDesignablesAgent;
     _opaque = NO;
+    _atLeastiOS_12_2_0 = [NSProcessInfo.processInfo isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){12,2,0}];
```

Comment:
Nit: The operating system version doesn’t change dynamically during the process’ lifetime, so we can set this variable in `+initialize` above. (The same also goes for `_isTargetingInterfaceBuilder`, come to think of it.)

Reply:
Yep, let's address this later.